set.mm

```diff
@@ -98481,15 +98481,15 @@ our Axiom of Choice (in the form of ~ ac2 ).  The proof does not depend
        (Contributed by NM, 26-Mar-2004.) $)
     kmlem4 $p |- ( ( w e. x /\ z =/= w ) ->
                     ( ( z \ U. ( x \ { z } ) ) i^i w ) = (/) ) $=
-      ( vy vv wel cv wne wa wn csn cdif cuni wral cin c0 wceq wi wal wcel neeq2
-      elequ1 anbi12d elequ2 notbid imbi12d eldif simpr eluni notbii alnex con2b
-      spv wex imnan eldifsn necom anbi2i bitri imbi1i 3bitr3i albii sylib sylbi
-      3bitr2i syl11 ralrimiv disj sylibr ) CAFZBGZCGZHZIZDCFZJZDVKAGZVKKLZMZLZN
-      VTVLOPQVNVPDVTEAFZVKEGZHZIZDEFZJZRZESZVNVPDGZVTTZWGVNVPRECWBVLQZWDVNWFVPW
-      KWAVJWCVMECAUBWBVLVKUAUCWKWEVOECDUDUEUFUMWJDBFZWIVSTZJZIZWHWIVKVSUGWOWNWH
-      WLWNUHWNWEWBVRTZIZEUNZJWQJZESWHWMWREWIVRUIUJWQEUKWSWGEWEWPJRWPWFRWSWGWEWP
-      ULWEWPUOWPWDWFWPWAWBVKHZIWDWBVQVKUPWTWCWAWBVKUQURUSUTVAVBVEVCVDVFVGDVTVLV
-      HVI $.
+      ( vy vv wel cv wne wa wn csn cdif cuni wral cin c0 wceq wi wal wcel eleq1
+      weq neeq2 anbi12d eleq2 notbid imbi12d eldif simpr wex eluni notbii alnex
```

Comment:
ahhh, I don't know what to do. elequ1/2 are more elementary than eleq1/2, so it would be more natural to use them, but it looks like the minimizer does not like them.

Reply:
Now the upper case section is shorter than the original by one byte

Reply:
It is strange the minimize_with didn't catch that shortening...

Reply:
No no, the minimizer caught it, I just forgot it.